### ISSUE TITLE
[@types/koa] BaseContext.assert function definition broken in 2.13.0

### ISSUE THREAD

**Issue by cduff** [DefinitelyTyped/DefinitelyTyped#51324]
Calls to BaseContext.assert are now resulting in the following error:

```
This expression is not callable.
  Type 'typeof assert' has no call signatures.ts(2349)
index.d.ts(29, 1): Type originates at this import. A namespace-style import cannot be called or constructed, and will cause a failure at runtime. Consider using a default import or import require here instead.
```

Looks to have been caused by [pull request #50249](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50249) by @pkuczynski.

**Comment by pkuczynski** [comment]
I will have a look at this shortly. Can you send me example piece of code, which I could add to the test suite?

**Comment by cduff** [comment]
> I will have a look at this shortly. Can you send me example piece of code, which I could add to the test suite?

Hi @pkuczynski, thanks for quick response. I'm not exactly sure how the the tests in the [test](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/koa/test) folder have been structured, however if you add something like

```
ctx.assert(true, 500, 'message');
```

anywhere it should reproduce the issue.

**Comment by pkuczynski** [comment]
We have `ctx.assert(true, 404, 'Yep!')` in our tests already and they don't fail.

 I will use the latest release of `@types/koa` in my other project and get back to you soon...

**Comment by cduff** [comment]
Hi @pkuczynski I think it might relate to the `esModuleInterop` setting in `tsconfig.json` as per [A namespace-style import cannot be called or constructed, and will cause a failure at runtime](https://stackoverflow.com/questions/49256040/a-namespace-style-import-cannot-be-called-or-constructed-and-will-cause-a-failu).

**Comment by pkuczynski** [comment]
Yeah, I know. The issue did not appeared in my original project as I have this turned on. I am trying to figure out why tests did not failed... will fire up PR soon...

**Comment by pkuczynski** [comment]
`import * as httpAssert from 'http-assert'` seems to be perfectly fine for importing `assert`. Does anybody else encounters this issue?

**Comment by pkuczynski** [comment]
Btw. which version of typescript are you using?

**Comment by pkuczynski** [comment]
@tellnes @orta andy suggestions?

**Comment by tellnes** [comment]
I would say just change it to `import * as httpAssert from 'http-assert'` as suggested above.

**Comment by pkuczynski** [comment]
@tellnes thats what I did:

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50249/files#diff-bb94074d9ae1164d531dd6eb17a8858f06f4129662469a7d5e1dd19cb5eed86cL28

Not sure why this would not work?

**Comment by cduff** [comment]
> Btw. which version of typescript are you using?

Latest i.e. 4.1.5.

I think replacing
```
import * as httpAssert from 'http-assert';
```
with 
```
import httpAssert from 'http-assert';
```
or reverting to what it was previously
```
import httpAssert = require('http-assert');
```
would work?

**Comment by tellnes** [comment]
@pkuczynski Ignore my last comment, I read it too fast. Reverting like @cduff is suggesting sounds like a good option, so I've approved https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51338

**Comment by cduff** [comment]
Hi @pkuczynski & @tellnes, thanks for addressing this issue. Do you know what else needs to happen for the fix to be released?

**Comment by pkuczynski** [comment]
As per https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51338#issuecomment-782213287

> A DT maintainer needs to approve changes which affect module config files

**Comment by cduff** [comment]
Thanks for fixing @pkuczynski! Fixed in 2.13.1.

**Comment by pkuczynski** [comment]
You welcome @cduff :)
